dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04200CreateDefaultVanityURL.java

```diff
@@ -0,0 +1,235 @@
+			
+			//Insert into Inode Table the reference of the field
+			dc.setSQL(INSERT_INODE_QUERY);
+			dc.addParam(DEFAULT_VANITY_URL_FIELDS_INODES[i]);
+			dc.addParam("");
+			dc.addParam(new Date());
+			dc.addParam("field");
+			dc.loadResult();
+			
+			//Insert into Field Table
+	        dc.setSQL(INSERT_FIELD_QUERY);
+	        dc.addParam(DEFAULT_VANITY_URL_FIELDS_INODES[i]);
+	        dc.addParam(field.get("structureInode"));
+	        dc.addParam(field.get("fieldName"));
+	        dc.addParam(field.get("fieldType"));
+	        dc.addParam(field.get("fieldRelationType"));
+	        dc.addParam(field.get("fieldContentlet"));
+	        dc.addParam(field.get("isRequired"));
+	        dc.addParam(field.get("isIndexed"));
+	        dc.addParam(field.get("isListed"));
+	        dc.addParam(field.get("velocityVarName"));
+	        dc.addParam(field.get("sortOrder"));
+	        dc.addParam(field.get("values"));
+	        dc.addParam(field.get("regexCheck"));
+	        dc.addParam(field.get("hint"));
+	        dc.addParam(field.get("defaultValue"));
+	        dc.addParam(field.get("isFixed"));
+	        dc.addParam(field.get("isReadOnly"));
+	        dc.addParam(field.get("isSearchable"));
+	        dc.addParam(field.get("isUnique"));
+	        dc.addParam(new Date());
+	        dc.loadResult();
+	        
+	        i++;
+		}
+		
+		
+	}
+
+	/**
+	 * This method creates each field that is required for the Vanity URL Content Type and return a list with them.
+	 * @return list of fields
+	 */
+	private List<Map<String, Object>> setBaseVanityURLFields() {
+		List fields = new ArrayList();
+		
+		fields.add(setFieldElements("Order", "com.dotcms.contenttype.model.field.TextField","", "integer2", true, true, false, "order", 5, "", "", "", "", true, false, true));
+		
+		fields.add(setFieldElements("Action", "com.dotcms.contenttype.model.field.SelectField","", "integer1", true, true, false, "action", 4, 
+				"200 - Forward|200\r\n301 - Permanent Redirect|301\r\n302 - Temporary Redirect|302\r\n401 - Auth Required|401\r\n403 - Auth Failed|403\r\n404 - Missing|404\r\n500 - Error|500", "", "", "", true, false, true));
+		
+		fields.add(setFieldElements("Forward To", "com.dotcms.contenttype.model.field.CustomField","", "text4", true, true, true, "forwardTo", 3, "$velutil.mergeTemplate('/static/content/file_browser_field_render.vtl')", "", "", "", true, false, false));
+		
+		fields.add(setFieldElements("Uri", "com.dotcms.contenttype.model.field.TextField","", "text3", true, true, true, "uri", 2, "", "", "", "", true, false, true));
+		
+		fields.add(setFieldElements("Site","com.dotcms.contenttype.model.field.CustomField","","text2", true, true, true,"site",1,"$velutil.mergeTemplate('/static/content/site_selector_field_render.vtl')","","","",true,false,false));
+		
+		fields.add(setFieldElements("Title", "com.dotcms.contenttype.model.field.TextField","", "text1", true, true, true, "title", 1, "", "", "", "", true, false, true));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 165 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)